java/src/jmri/jmrit/dispatcher/TrainInfoFile.java

```diff
@@ -98,6 +98,18 @@ public TrainInfo readTrainInfo(String name) throws org.jdom2.JDOMException, java
                     } else {
                         log.error("Transit name missing when reading TrainInfoFile {}", name);
                     }
+                    if (traininfo.getAttribute("dynamictransit") != null ) {
+                        tInfo.setDynamicTransit(false);
+                        if (traininfo.getAttribute("dynamictransit").getValue().equals("yes")) {
+                            tInfo.setDynamicTransit(true);
+                        }
+                    }
+                    if (traininfo.getAttribute("dynamictransitcloseloop") != null ) {
+                        tInfo.setDynamicTransitCloseLoopIfPossible(false);
+                        if (traininfo.getAttribute("dynamictransitcloseloop").getValue().equals("yes")) {
+                            tInfo.setDynamicTransitCloseLoopIfPossible(true);
+                        }
```

Comment:
The comment above applies to this section as well.

Reply:
All the infra structure is in place but its not implemented.